src/main/java/com/suse/salt/netapi/calls/modules/Test.java

```diff
@@ -138,6 +138,12 @@ public VersionInformation(Map<String, String> salt,
         return PING;
     }
 
+    public static LocalCall<Boolean> ping(Optional<Integer> timeout,
```

Comment:
For the record, we discussed this with @meaksh and he will adjust this PR as per your suggestion. Thanks